addons/web/static/src/core/notifications/notification.scss

```diff
@@ -1,48 +1,23 @@
-$notification-width: 320px;
-$notification-text-color: #212529;
-$notification-title-text-color: #6c757d;
-
 // Colors of the notification are not managed by this file.
 // It is handled by the other classes defining a background. (like danger, warning, etc.)
 // The notification just add overlays.
 
 .o_notification_manager {
-  @include o-position-absolute(3.5rem, 0);
-  position: fixed;
+  @include o-position-absolute(4.25rem, 0);
   z-index: ($zindex-modal + $zindex-popover) / 2;
-  width: $notification-width;
-  margin-right: 0.5rem;
-  max-width: 100%;
+  width: $o-notification-width;
+  right: 0.5rem;
 
   .o_notification {
-    width: 100%;
     margin-bottom: 0.375rem;
-    overflow: hidden;
-    background-color: rgba(255, 255, 255, 0.7);
     background-clip: padding-box;
-    border: 1px solid rgba(0, 0, 0, 0.1);
-    box-shadow: 0 0.25rem 0.75rem rgba(0, 0, 0, 0.1);
-    border-radius: 0.25rem;
-
-    .o_notification_body {
-      padding: 0.5rem 0.625rem;
-      background-color: rgba(255, 255, 255, 0.78) !important;
-      color: $notification-text-color;
-
-      .o_notification_close {
-        margin-left: 1rem;
-      }
-    }
+    box-shadow: 0 0.25rem 1rem 0 rgba(0, 0, 0, 0.1), inset 0px 0px 0px 1px $o-gray-400;;
+  }
 
-    .o_notification_header {
-      display: flex;
-      align-items: center;
-      padding: 0.5rem 1.5rem;
-      color: $notification-title-text-color;
-      background-color: rgba(255, 255, 255, 0.7);
-      border-bottom: 1px solid rgba(0, 0, 0, 0.05);
-    }
+  .toast-header, .toast-body {
+    border-left-width: $o-notification-border-width !important;
```

Comment:
```scss
border: 0 0 0 $o-notification-border-left-width;
```
Doing so you don't have to add all those border classes. I guess that the reason was to reduce CSS but you'll have to add this line anyway.

Also, let's try to name variables according to their use... in this specific case is probably debatable but I'd suggest to add `-left-` to the variable name.